editor/editor_node.cpp

```diff
@@ -1001,6 +1001,12 @@ void EditorNode::_resources_reimported(const Vector<String> &p_resources) {
 		}
 	}
 
+	// Editor may crash when related animation is playing while re-importing GLTF scene, stop it in advance.
+	AnimationPlayer *ap = AnimationPlayerEditor::get_singleton()->get_player();
+	if (ap != nullptr) {
```

Comment:
```suggestion
	if (ap) {
```
他のほとんどのコードはこれで nullptr をチェックします。

Reply:
> Is there anything different between 0 and 1? If there is no problem to call that stop() on both, then this check would not be necessary anyway.

I did comment above though. I missed your comment, does this mean that the animation does not stop when the image is re-imported, etc.? If so, I think `if (scenes.size() > 0)` check is available and required.

Reply:
> does this mean that the animation does not stop when the image is re-imported, etc.?

Yes.